airflow/script/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+CMD="airflow"
```

Comment:
Ideally, we simply extract these from airflow.cfg, rather then needing to define them both in the config and as environment variables to the container. Requires a little chopping voodoo on the strings in the cfg as they are not in this format. Also, it looks like these are hard coded, and that if given over the environment (via k8s pod) they would be ignored.